### PR TITLE
i18n(ko-KR): create `asides.mdx` and update `authoring-content.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description

- create `asides.mdx`
- update `authoring-content.mdx`

### CODE REVIEW

File: docs/src/content/docs/ko/components/asides.mdx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    // 예시 코드 스니펫.
+    ```
+
+    </Aside>
+
+    <Aside type="danger">Do not give your password to anyone.</Aside>
+
+</Fragment>
+
+</Preview>
```
[the remark this answers is not in the document]
Comment (jsparkdev) [reply docs/src/content/docs/ko/components/asides.mdx]: Thank you, I have translated all of content.

File: docs/src/content/docs/ko/components/asides.mdx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+</Fragment>
+
+<Aside slot="preview" type="caution" title="Watch out!">
+	A warning aside *with* a custom title.
+</Aside>
+
+</Preview>
+
+## `<Aside>` Props
```
[the remark this answers is not in the document]
Comment (jsparkdev) [reply docs/src/content/docs/ko/components/asides.mdx]: I felt this needed to be translated and have done so, thank you.

File: docs/src/content/docs/ko/components/asides.mdx [hunk 3]
```diff
@@ -0,0 +1,158 @@
+---
+title: Asides
```
[the remark this answers is not in the document]
Comment (jsparkdev) [reply docs/src/content/docs/ko/components/asides.mdx]: I had a lot of trouble thinking about this, so I got some help from the AI and finally decided to use 보조 내용! Thanks :)

**Review by HiDeoo — APPROVED** [review]
Amazing, thanks for quickly addressing the suggestions, everything looks good to me :rocket:

**Pull request merged**